example-playbooks/replace_node/replace_node.yml

```diff
@@ -127,12 +148,31 @@
     - ansible-scylla-node
 
 
+# Update scylla-monitoring before the replacement starts to be able to see the
+# streaming progress
```

Comment:
Good comment but this change doesn't belong to this patch.
Send it as a separate patch, please.

Reply:
Done